packages/amplify-graphql-conversation-transformer/src/resolvers/invoke-lambda-resolver-fn.js

```diff
@@ -0,0 +1,56 @@
+import { util } from '@aws-appsync/utils';
+
+export function request(ctx) {
```

Comment:
For future PR:
1. Please add a doc comment noting the replacement logic
2. Rename the file to something other than `.js` (e.g., `.js.template` or something), since this isn't actually executable JS as-is

Reply:
Good call. 
I addressed 2. in b05fbac4e071750d75b5c3523546ca0aa2245ed9 because it's core to the scope of change.
I'll follow up for 1. in https://github.com/aws-amplify/amplify-category-api/pull/2897#discussion_r1777331065